File: 백준/Bronze/10807. 개수 세기/개수 세기.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;
import java.util.HashSet;
import java.util.Set;
import java.util.StringTokenizer;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));

        int size = Integer.parseInt(br.readLine());

        int[] seq = new int[size];
        StringTokenizer st = new StringTokenizer(br.readLine());
        for (int i = 0; i < size; i++) {
            seq[i] = Integer.parseInt(st.nextToken());
        }

        int target = Integer.parseInt(br.readLine());
        int cnt = 0;

        for (int num : seq) {
            if (num == target) {
                cnt++;
            }
        }
        System.out.print(cnt);
    }
}

File: 백준/Bronze/10807. 개수 세기/개수 세기.kt
import java.io.BufferedReader
import java.io.InputStreamReader

fun main() = with(BufferedReader(InputStreamReader(System.`in`))) {
    val n = readLine().toInt()
    val arr = readLine().split(" ").map { it.toInt() }.toIntArray()
    var v = readLine().toInt()
    var cnt = 0
    for (i in arr) {
        if (v + 100 == i + 100) {
            cnt++
        }
    }
    println(cnt)
}